### ISSUE TITLE
Design Intent - Accessibility Feedback - MHV to VA.gov Team - Medical Records - Immunizations

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#51727]
## VFS acceptance criteria
- [ ] Let Platform know when the **Must** feedback has been incorporated 
- [ ] Leave any comments for feedback you decide _not_ to take
- [ ] VFS team closes the ticket

## Thoughts/questions
- You're asking/thinking about accessibility considerations right from the start, which I appreciate so much! Thank you!

## Feedback
- Practice areas will document their feedback following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback).

**Must:**
- I mentioned this in the meeting, but I've confirmed that the Print > Save as PDF browser workflow does not result in an accessible PDF, even if the source material is fully accessible. I don't have a strong opinion on including both Download and Print or just Download, but I don't think Print alone is going to be a viable option if the intention is to allow users to download a PDF copy for their own records.

**Should:**
- Now is a great time to think about what kind of `aria-label` might be appropriate for the View Details links. Each link should have a unique accessible name (either the link text itself is unique or it has a unique `aria-label`). It will probably depend on what kind of vaccination metadata you can get, but something like "View details for [type of vaccine] on [date]" would probably be sufficient.
- I would recommend against sticky buttons at the bottom of the Print VA Vaccines screen. Sticky elements can cause issues for screen magnification/high zoom users, and can also be confusing/add cognitive load for sighted keyboard users (visually unclear what the [Tab] sequence is to get to those buttons). It's not a hard no, but it is something you'll want to test to validate that the benefits of the sticky element are worth any tradeoffs and the accessibility issues are mitigated as much as possible. (For what it's worth, [the Gov.UK team decided it wasn't worth it](https://technology.blog.gov.uk/2018/05/21/sticky-elements-functionality-and-accessibility-testing/) -- but your mileage may vary!)
   - If you do want to go with a sticky element, it would be very helpful at your Midpoint Review if you have a design mockup at 200%, 300%, and 400% zoom. Doesn't have to be a fully-completed prototype, just something to verify that the product will still be usable at high zoom.

**Consider:**
- For the Print VA Vaccines view, consider whether the View Details link is required once users reach this screen. If a keyboard-only user has received a lot of vaccinations (let's say one flu shot per year for 30 years) and they want to download/print the two most recent records, then they'll have to [Tab] through all of the checkboxes and details links (56 keystrokes) to get to the print button. Definitely not ideal! If the View Details link isn't necessary for this screen, then at minimum you cut the [Tab] stops in half, and maybe you can even simplify down into a modified [va-checkbox-group component](https://design.va.gov/components/form/checkbox) which makes skipping past the rest of the fieldset even easier.
   
   If it's important to keep the View Details link, we still might be able to find a way to simplify this for keyboard users. Just let me know if you want to chat about this sometime.

## Platform directions
- Remove the governance-team label
- Update "Issue Title"
- Link to collab cycle Request epic
- Add your feedback
- Remove the CC-Request label (only if you created the ticket using the New Issue link from the Collaboration Cycle Reviews board)
- Add assignees based on collab cycle touchpoint
  - **Design Intent**: VFS designer, VFS PM (optional), yourself (optional)
  - **Midpoint Review**: VFS PM, yourself (optional)
  - **Staging Review QA Only**: VFS PM, yourself (optional)

**Comment by 12310177** [comment]
I Tonya Walker a navy veteran has been through a horrible experience. With the Carl Vinson VA hospital in Dublin ga. The community care team and sheineka. I mad at if I cancel a appointment. Was told that I can cancel long as I reschedule. Then I asked for a doctor in my community. Was told that a woman named sheneika says that if I cancel a appointment the VA won't pay for my care. And I must call her at all times. I have a new doctor in my hometown. Was told that I have two referral for my medical problem that I have been having was cancelled out from the community care team and sheineka. I had the Whitehouse hotline to call Mrs sheneika and to straighten out this matter. About me receiving my medical care in my community. She had called and says she will put the referral in for me to see a doctor of my choice. I was told again that I do not have the referral for my doctor of my choice and I must go where they says to go. For my medical needs. Fix matter. Carl Vinson VA hospital in Dublin ga and sheineka of the community care team. Keep me informed

**Comment by 12310177** [comment]
***@***.***
